working/augmentation-libraries/feature-specification.md

```diff
@@ -247,86 +260,90 @@ To help avoiding issues, we want to introduce a *lint* which warns if a library
+*   **Augmenting fields**: Within an augmenting variable declaration,
+    `augmented` can only be used in an initializer expression, and refers
+    to the augmented variable's initializing expression, which is
+    immediately evaluated.
 
-    It is a compile-time error to use `augmented` in an augmenting field's
-    initializer if the member being augmented is not a field with an
-    initializer.
+    It is a compile-time error to use `augmented` in an augmenting
+    variable's initializer if the member being augmented is not a variable
+    declaration with an initializing expression.
 
-*   **Augmenting functions**: When augmenting a function, `augmented` refers to
-    the augmented function. Tear offs are not allowed, so this function must
-    immediately be invoked.
+*   **Augmenting functions**: When augmenting a function, `augmented`
+    refers to the augmented function. Tear offs are not allowed, so this
+    function must immediately be invoked.
 
-*   **Augmenting operators**: When augmenting an operator, `augmented` refers
-    to the augmented operator method, which must be immediately invoked using
-    function call syntax. For example when augmenting `operator +` you would use
-    `augmented(1)` to call the augmented operator, and when augmenting
-    `operator []=` you would use the `augmented(key, value)` syntax.
+*   **Augmenting operators**: When augmenting an operator, `augmented`
+    refers to the augmented operator method, which must be immediately
+    invoked using function call syntax. For example, when augmenting
+    `operator +` you could use `augmented + 1` to call the augmented
+    operator, and when augmenting `operator []=` you would use the
+    `augmented[key] = value` syntax.
 
-    *   Note that `augmented` in such an augmenting operator method body is not
-        an expression by itself, and cannot be used to tear off the augmented
-        operator method. Similar to `super`, it is a syntactic form which can
-        only be used in limited ways.
+    *   Note that `augmented` in such an augmenting operator method body is
+        not an expression by itself, and cannot be used to tear off the
+        augmented operator method. Similar to `super`, it is a syntactic
+        form which can only be used in limited ways.
 
-*   **Augmenting enum values**: When augmenting an enum value, `augmented` has
-    no meaning and is not allowed.
+*   **Augmenting enum values**: When augmenting an enum value, `augmented`
+    has no meaning and is not allowed.
 
 In all relevant cases, if the augmented member is an instance member, it is
 invoked with the same value for `this`.
 
-Assume that the identifier `augmented` occurs such that the outermost
-enclosing declaration is not an augmenting declaration. In this case, the
-identifier is taken to be a reference to a declaration which is in scope.
+Assume that the identifier `augmented` occurs such that no enclosing
```

Comment:
Maybe "such that" -> "in a source location where".

Reply:
Done.